GroundSensor Left
GroundSensor Center
GroundSensor Right